Documentation/Benchmarks.md

```diff
@@ -80,6 +80,75 @@ iPhone 16 Pro Max run, and only for models that were reloaded during the session
 | Decoder       |                       88.49 |                        8.11 |              146.01 | MLComputeUnits(rawValue: 1) |
 | JointDecision |                       48.46 |                        7.97 |               71.85 | MLComputeUnits(rawValue: 1) |
 
+## Text-to-Speech
+
+### Kokoro-82M PyTorch Pipeline
+
+```bash
+uv run python benchmark_kokoro_pip.py
```

Comment:
oh also, this is probably not a fair apples to apples comparison since the torch is using "cpu" as the backend. It might be a lot faster with "mps" as the backend. Might even be almost on par with MLX 

EIther copy the scripts here or provide some documentation on what thtese scripts are doing?

Reply:
as discussed, just leave this to me after you merge, I will run the scripts on my laptop